docs/modules/ROOT/partials/getting-started/proc-configuring-registry-security.adoc

```diff
@@ -47,7 +47,9 @@ endif::[]
 ** *Client Protocol*: `openid-connect`
 ** *Access Type*: `bearer-only`
 + 
-You can use the defaults for the other client settings.  
+You can use the defaults for the other client settings. 
++
+NOTE: If you are using OpenShift service accounts, the client *Access Type* must be `confidential` instead of `bearer-only`. 
```

Comment:
Sorry @smccarthy-ie, I should make it more clear. They're not Openshift service accounts. The note should be about users that want to use Keycloak service accounts.

Reply:
Aha! OK, sorry, no problem :)